terraform/aws/modules/security/vpc_flow_logs.tf

```diff
@@ -5,23 +5,10 @@ data "aws_kms_alias" "s3" {
 }
 
 resource "aws_flow_log" "cloud_vpc" {
-  count                = length(data.aws_vpcs.vpcs.ids)
-  log_destination      = aws_s3_bucket.vpc_flow_logs.arn
+  count                = var.enable_flow_logs ? length(data.aws_vpcs.vpcs.ids) : 0
+  log_destination      = "arn:aws:s3:::mm-cloud-flowlogs-${var.environment}"
```

Comment:
@angeloskyratzakos Have we created these buckets for each of our environments in the security account?

Reply:
Daniel has already created them, except Dev env, where we will enable it in the future